doma-core/src/test/java/org/seasar/doma/jdbc/builder/DeleteBuilderTest.java

```diff
@@ -49,4 +55,78 @@ public void testLiteral() {
 
     builder.execute();
   }
+
+  @Test
+  public void testParams() throws Exception {
+    UpdateBuilder builder = UpdateBuilder.newInstance(new MockConfig());
```

Comment:
I think `DeleteBuilder` is correct in this test class.

Reply:
I apologize.
I just fixed now.